internal-api/src/main/proto/systems/ajax/malov/internalapi/input/reqreply/stock/getFiveBestStockSymbolsWithRecords.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+import "systems/ajax/malov/internalapi/commonmodel/stock/short_stock.proto";
+import "systems/ajax/malov/internalapi/commonmodel/stock/big_decimal.proto";
+
+
+package systems.ajax.malov.input.reqreply.stock.get_five_best_stock_symbols_with_stocks.proto;
+
+option java_multiple_files = true;
+
+message GetFiveBestStockSymbolsWithStockRecordsRequest {
+
+}
```

Comment:
not extendable API. probably it's better to pass how many TOP stocks you want to receive as request parameter